eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/RouteBlinding.scala

```diff
@@ -66,26 +118,73 @@ object RouteBlindingEncryptedDataCodecs {
   private val outgoingNodeId: Codec[OutgoingNodeId] = (("length" | constant(hex"21")) :: ("node_id" | publicKey)).as[OutgoingNodeId]
   private val pathId: Codec[PathId] = variableSizeBytesLong(varintoverflow, "path_id" | bytes).as[PathId]
   private val nextBlinding: Codec[NextBlinding] = (("length" | constant(hex"21")) :: ("blinding" | publicKey)).as[NextBlinding]
+  private val paymentRelay: Codec[PaymentRelay] = (
+    ("length" | constant(hex"0a")) ::
+      ("fee_base_msat" | millisatoshi32) ::
+      ("fee_proportional_millionths" | uint32) ::
+      ("cltv_expiry_delta" | cltvExpiryDelta)).as[PaymentRelay]
```

Comment:
The codec for `payment_relay` has been changed based on feedback by Rusty, the spec says it should be:

```
    1. type: 10 (`payment_relay`)
    2. data:
        * [`u16`:`cltv_expiry_delta`]
        * [`u32`:`fee_proportional_millionths`]
        * [`tu32`:`fee_base_msat`]
```

The reason for that is to be more future-proof in case we want to remove the base fee in the future.
Can you also add a test to verify we correctly encode/decode a base fee of 0 msat (which shouldn't use any byte)?

Reply:
Done, there was already a test with a base fee of 0 and it works as expected, taking 0 byte.